docs/source/posts/2021/2021-07-12-week-6-sajag.rst

```diff
@@ -0,0 +1,44 @@
+Fifth week of coding!
+=====================
+
+.. post:: July 12 2021
+   :author: Sajag Swami
+   :tags: google
+   :category: gsoc
+
+Welcome to the sixth weekly check-in. I'll be sharing my progress for the fifth week of coding.
+
+What did you do this week?
+--------------------------
+
+1. Generalised the vtkProteinRibbonFilter implementation.
```

Comment:
typo: Generalized

Reply:
Generalise is correct, it's the British spelling.